.github/workflows/ansible-push-release-tag.yml

```diff
@@ -0,0 +1,39 @@
+name: Push Ansible Release Tag
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - deployments/ansible/galaxy.yml
+
+jobs:
+  push-ansible-release-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Read current version of the Ansible Collection
+        id: read-galaxy-yaml
+        uses: cumulusds/get-yaml-paths-action@v0.1.0
+        with:
+          file: deployments/ansible/galaxy.yml
+          version: version
+
+      - name: Push Ansible Release Tag
+        id: check-tag
+        run: >
+          git tag ${RELEASE_TAG} 2>/dev/null 
+            && git push "${REMOTE_REPO}" --tags
+            || echo "tag already exists"
+        env:
+          RELEASE_TAG: ansible-v${{ steps.read-galaxy-yaml.outputs.version }}
+          REMOTE_REPO: https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${REPOSITORY}.git
```

Comment:
I think GDI spec (and security guidelines) force us to use GITHUB_TOKEN with "readonly" permissions. 

Which repos are we pushing to here and need write access to as a result?

Reply:
same repo https://github.com/signalfx/splunk-otel-collector. it uses the temporary github action's token

Reply:
We do the same in https://github.com/signalfx/splunk-otel-collector-chart